src/connection.ts

```diff
@@ -210,6 +211,36 @@ export class Connection extends JSForceConnection {
     return super._baseUrl();
   }
 
+  /**
+   * Will deploy a recently validated deploy request
+   *
+   * @param options.id = the id of the deploy to quickly validate
```

Comment:
This param description isn't quite accurate.  It's the deploy ID that's been validated already from a previous checkOnly deploy request.   See this doc for lots of details: https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_deployRecentValidation.htm